### PR TITLE
ELY-2379 Rename a local variable in FileAuditEndpoint class [local_variable_renamed]

### PR DESCRIPTION
**Pull request by OluchukwuCatherineObi-Njoku**
https://issues.redhat.com/browse/ELY-2379

### CODE REVIEW

File: audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java [hunk 1]
```diff
@@ -70,22 +70,22 @@ public class FileAuditEndpoint implements AuditEndpoint {
     }
 
     void setFile(final File file) throws IOException {
-        boolean ok = false;
+        boolean isSetUp = false;
```
Comment (skyllarr) [inline audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java]: @OluchukwuCatherineObi-Njoku Please rename to for example `isFileSet` or `fileSet` to be more specific about what has been set

File: audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java [hunk 2]
```diff
@@ -108,7 +108,7 @@ private void safeClose(Closeable c) {
      * This method can be overridden by subclasses to modify data written into file (to encrypt them for example),
      * or just for counting amount of written bytes for needs of log rotation and similar.
      *
-     * This method can be invoked only in synchronization block surrounding one log message processing.
+     * This method can be invisSetUped only in synchronization block surrounding one log message processing.
```
Comment (skyllarr) [inline audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java]: @OluchukwuCatherineObi-Njoku This should not have been renamed, same goes for line 124

**Review by skyllarr — COMMENTED** [review]
Hi @OluchukwuCatherineObi-Njoku , thank you for the PR! I added some feedback

**Comment by PrarthonaPaul** [comment]
Hi @OluchukwuCatherineObi-Njoku 
Just a reminder, there are some comments on this PR that need to be addressed. 
If you have any questions about any of the comments, please feel free to let us know :)

File: audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java (hunk 1 above)
Comment (OluchukwuCatherineObi-Njoku) [reply audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java]: Hello. I made the suggested updates. Thanks

Comment (OluchukwuCatherineObi-Njoku) [reply audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java]: > Hello. I made the suggested updates. Thanks

@Skyllarr

File: audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java [hunk 3]
```diff
@@ -121,7 +121,7 @@ void write(String toWrite) throws IOException {
      * This method is NO-OP by default. It is intended to be overridden by subclasses
      * which need to perform some operation before every writing into the target local file.
      *
-     * This method can be invoked only in synchronization block surrounding one log message processing.
+     * This method can be invisFileSeted only in synchronization block surrounding one log message processing.
```
Comment (fjuma) [inline audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java:124]: This shouldn't be renamed.

**Review by fjuma — CHANGES_REQUESTED** [review]
Thanks for the updates @OluchukwuCatherineObi-Njoku! Just one more spot where a word shouldn't be updated.

**Comment by darranl** [comment]
Thank you for your PR FYI I have just submitted https://github.com/wildfly-security/wildfly-elytron/pull/2112 to correct the comment update that crept in with this PR.

**Pull request closed without merging**